scrapyrt/core.py

```diff
@@ -164,7 +164,7 @@ def _get_log_file_path(self):
         log_dir = os.path.join(self.log_dir, self.spider_name)
         if not os.path.exists(log_dir):
             os.makedirs(log_dir)
-        filename = datetime.datetime.now().isoformat() + '.log'
+        filename = datetime.datetime.now().strftime('%Y-%m-%dT%H;%M;%S.%f') + '.log'
```

Comment:
Looking into ISO standard we can also have date written like this:
```
20161208T102859Z
```

or like this 
```
 2007-04-05T1430Z
```
I think 2nd option looks good, should work fine in windows and is fairly human readable.